components/AmbulanceCard.jsx

```diff
@@ -90,7 +91,8 @@ const AmbulanceCard = ({
           <hr />
           <div className="flex justify-between items-center mx-2 mt-2 pb-3">
             <div className="font-mono text-gray-700 text-sm dark:text-white">
-              {parseDateString(createdTime)}
+              {lastVerifiedOn ?
+                              `Verified @ ${parseDateString(lastVerifiedOn)}` : null}
```

Comment:
More better way to  condition it: 
```jsx
{lastVerifiedOn && `Verified @ ${parseDateString(lastVerifiedOn)}` }
```

<img width="992" alt="Screenshot 2021-04-24 at 12 09 00 AM" src="https://user-images.githubusercontent.com/61158210/115915682-492f9380-a491-11eb-9205-572185f9f339.png">

Reply:
@Deep-Codes modified. plz review again.